<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowPreference</class>
 <widget class="QDialog" name="WindowPreference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>191</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>191</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>461</width>
    <height>191</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Helena - Preferências</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/main.qrc">
    <normaloff>:/iconPower.ico</normaloff>:/iconPower.ico</iconset>
  </property>
  <widget class="QLabel" name="label_pathDatabase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>185</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Path da base de dados:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_tableWires">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>185</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome da tabela dos fios:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_tableBobbins">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>185</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome da tabela dos carreteis:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tableWires">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pathDatabase">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_tableLaminas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>185</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome da tabela das lâminas:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tableLaminas">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tableBobbins">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tableTransformers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>185</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome da tabela dos transformadores:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tableTransformers">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icons/main.qrc"/>
 </resources>
 <connections/>
</ui>
